src/test/java/seedu/address/logic/parser/FindCommandParserTest.java

```diff
@@ -57,6 +58,16 @@ public void parse_validArgs_returnsFindCommand() {
     }
 
     // Find Descriptor tests
+    @Test
+    public void findDescriptor_validPrefixInput_addToPredicateList() throws ParseException {
+        String validExperiencePrefixInput = " y/3";
+        ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(validExperiencePrefixInput,
```

Comment:
As Chen En mentioned in the group, I believe the method used should be `tokenizeWithoutPreamble` since we are doing `FindCommandParser` tests

Reply:
👍